### PR TITLE
RHBPMS-4021 - Not able to open BP without browser refresh

### PR DESCRIPTION
**Pull request by tsurdilo**

### CODE REVIEW

**Comment by tsurdilo** [comment]
depends on https://github.com/uberfire/uberfire/pull/419

File: jbpm-designer-client/src/main/java/org/jbpm/designer/client/DesignerPresenter.java [hunk 1]
```diff
@@ -179,7 +179,9 @@ public void notifyOpenInXMLEditor( @Observes final DesignerNotificationEvent eve
     }
 
     public void onLockChange( @Observes UpdatedLockStatusEvent updateLockEvent ) {
-        view.raiseEventUpdateLock();
+        if(updateLockEvent.getFile().getFileName().equals(versionRecordManager.getCurrentPath().getFileName())) {
```
Comment (manstis) [inline jbpm-designer-client/src/main/java/org/jbpm/designer/client/DesignerPresenter.java]: What if you have two files with the same name in two different packages?

**Comment by csadilek** [comment]
The build failed as the branch names don't line up @tsurdilo. No required to fix that now, but in the future please make sure dependent PR's have a feature/bug branch with the same name. Simply amend your commit with the changes discussed then we can merge this.

**Comment by tsurdilo** [comment]
@csadilek amended

**Comment by csadilek** [comment]
+1

**Pull request merged**